goshopify.go

```diff
@@ -343,21 +365,30 @@ func (c *Client) doGetHeaders(req *http.Request, v interface{}) (http.Header, er
 }
 
 func wrapSpecificError(r *http.Response, err ResponseError) error {
-	if err.Status == 429 {
-		f, _ := strconv.ParseFloat(r.Header.Get("retry-after"), 64)
+	// see https://www.shopify.dev/concepts/about-apis/response-codes
+	if err.Status == http.StatusTooManyRequests {
+		f, _ := strconv.ParseFloat(r.Header.Get("Retry-After"), 64)
 		return RateLimitError{
 			ResponseError: err,
 			RetryAfter:    int(f),
 		}
 	}
-	if err.Status == 406 {
-		err.Message = "Not acceptable"
+
+	// if err.Status == http.StatusSeeOther {
+	// todo
```

Comment:
Should we address this todo in the current PR?

Reply:
well we were talking about different statuses to redirects on and stumbled across the list of api codes and this one came up.. thought it was notable to at least address it's existence. it's not really in scope of a retry imo, we could add a "WithFollow" option or something which acknowledges these?